sledgehammer/start-up.sh

```diff
@@ -128,12 +128,16 @@ MYIP="${BASH_REMATCH[1]}"
 if suse_ver 12; then
     [ -f /etc/sysconfig/network/config ] && source /etc/sysconfig/network/config
     WAIT_FOR_INTERFACES=${WAIT_FOR_INTERFACES:-120}
-    /usr/lib/wicked/bin/wickedd-dhcp$ip_version --test --test-output /tmp/wicked-dhcp-$BOOTDEV --test-timeout $WAIT_FOR_INTERFACES $BOOTDEV
+    /usr/lib/wicked/bin/wickedd-dhcp$IP_VERSION --test --test-output /tmp/wicked-dhcp-$BOOTDEV --test-timeout $WAIT_FOR_INTERFACES $BOOTDEV
     source /tmp/wicked-dhcp-$BOOTDEV
     if [ -z "$ADMIN_IP" ]; then
         ADMIN_IP=$SERVERID
     fi
-    DOMAIN=$DNSDOMAIN
+    if [ -z "$DNSDOMAIN" ]; then
+        DOMAIN=${DNSSEARCH%%,*}
```

Comment:
I understand what this line does, but I'd very much prefer a short comment to save future confusion.

Reply:
Done